Stats and sales for Gatling Turrets (Single & Quad) (#2)
TheGiraffe3/rumskib: data/sales.txt

outfitter "Kraz Advanced"
	"Quad Beam Laser"

outfitter "Remnant"
	"Inhibitor Turret"

outfitter "South Pirate Advanced"
	"Rumskib Single Gatling Turret"

> mission "Rumskib Add Quad Gat Turret"
> 	outfitter
> 	invisible
> 	source
> 		attributes "pirate"
> 	to offer
> 		has "event: war begins"
> 	on offer
> 		event "rumskib add quad gat turret"

Reviewer: ```suggestion
mission "Rumskib: Add Quad Gat Turret"
	outfitter
	invisible
	source
		attributes "pirate"
	to offer
		has "event: war begins"
	on offer
		event "rumskib: add quad gat turret" 0
```

		conversation
			scene "outfit/gat turret quad"

> 			`As you watch an one-eyed captain proudly polishing a new Gatling turret with four barrels, it occurs to you that the Pirates must have joined the arms race.`

Reviewer: This sentence doesn't feel.... right, somehow. Do you mind if I rework it, something like this?
```suggestion
			`When you walk into the outfitter, you notice the proprietor proudly polishing the quadruple barrels of a new Gatling Turret. "Ho, Captain!" he calls. "Come buy the new Quad Gatling Turret! Only one hundred and eighty thousand credits, for enough firepower to destroy any hostile ship!"`
			`	The pirates must have joined the arms race when the rebellion started.`
```

				decline

> event "rumskib add quad gat turret"

Reviewer: ```suggestion
event "rumskib: add quad gat turret"
```

outfitter "Kraz Advanced"
	"Quad Beam Laser"

outfitter "Remnant"
	"Inhibitor Turret"

outfitter "South Pirate Advanced"
	"Rumskib Single Gatling Turret"

mission "Rumskib: Add Quad Gat Turret"
	outfitter
	invisible
	source
		attributes "pirate"
	to offer
		has "event: war begins"
	on offer
		event "rumskib: add quad gat turret" 0
		conversation
			scene "outfit/gat turret quad"
			`When you walk into the outfitter, you notice the proprietor proudly polishing the quadruple barrels of a new Gatling Turret. "Ho, Captain!" he calls. "Come buy the new Quad Gatling Turret! Only one hundred and eighty thousand credits, for enough firepower to destroy any hostile ship!"`
			`	The pirates must have joined the arms race when the rebellion started.`
				decline

event "rumskib: add quad gat turret"
	outfitter "South Pirate Advanced"

> 		add "Gatling Turret (Quad)"

Reviewer: ```suggestion
		add "Rumskib Quad Gatling Turret"
```
